ravenframework/Optimizers/RavenSampled.py

```diff
@@ -150,6 +152,19 @@ def handleInput(self, paramInput):
     # additional checks
     if self.limit is None:
       self.raiseAnError(IOError, 'A <limit> is required for any RavenSampled Optimizer!')
+    #These objective multipliers are use so that the objective can always
+    # appear as a minimization problem internally.
+    # This multipiles by -1 to turn a maximization problem to a minimization
+    # problem.
+    self._objMult = {} #max will be -1, min will be 1
+    self._objMultArray = np.ones(len(self._objectiveVar))
```

Comment:
Please move these definitions in "__init__" method

Reply:
Fixed in 545dd17eee14f01d61d82180864382c88b5144e5